package/cpp/rnskia/dom/nodes/JsiShaderNodes.h

```diff
@@ -328,12 +346,12 @@ class JsiBaseGradientNode : public JsiBaseShaderNode {
     }
   }
 
-  SkColor *_colors;
-  SkTileMode _mode;
+  const SkColor *_colors;
   double _flags;
-  SkScalar *_positions;
-  SkMatrix *_matrix;
   int _colorCount;
+  SkTileMode _mode;
+  const SkScalar *_positions;
+  const SkMatrix *_matrix;
```

Comment:
It was done so that the base gradient node can read and propagate these values, and then subclasses use them - since gradient nodes shares these props. Just a simplification. :)